docs/user/security/authentication/index.asciidoc

```diff
@@ -145,43 +194,52 @@ xpack.security.authc.saml.maxRedirectURLSize: 1kb
 Similar to SAML, authentication with OpenID Connect allows users to log in to {kib} using an OpenID Connect Provider such as Google, or Okta. OpenID Connect
 should also be configured in {es}. For more details, see {ref}/oidc-guide.html[Configuring single sign-on to the {stack} using OpenID Connect].
 
-Set the configuration values in `kibana.yml` as follows:
+Enable the OpenID Connect authentication specifying which OpenID Connect realm in {es} should be used:
 
-. Enable the OpenID Connect authentication:
-+
 [source,yaml]
 --------------------------------------------------------------------------------
-xpack.security.authc.providers: [oidc]
+xpack.security.authc.providers:
+  oidc.oidc1:
+    order: 0
+    realm: oidc1
 --------------------------------------------------------------------------------
 
-. {kib} needs to specify which OpenID Connect realm in {es} should be used, in case there are more than one configured there.
-+
-[source,yaml]
---------------------------------------------------------------------------------
-xpack.security.authc.oidc.realm: oidc1
---------------------------------------------------------------------------------
+If you want to use Third Party initiated Single Sign-On, configure your OpenID Provider to use `/api/security/oidc/initiate_login` as `Initiate Login URI`.
+
+It's also possible to configure multiple OpenID Connect authentication providers at the same time. In this case users will need to choose which provider to use for login at the Login Selector screen:
```

Comment:
`In this case users will need to choose which provider to use for login at the Login Selector screen:` > `In this case, you need to choose which provider to use for login at the Login Selector UI:`